README.md

```diff
@@ -1,2 +1,108 @@
-# To-Do-List-with-Webpack
-An interactive To-do list with Webpack
+<a name="readme-top"></a>
+
+# To-Do List Project
+
+# 📗 Table of Contents
+
+- [📖 About the To-Do List Project](#about-project)
+- [🛠 Built With](#built-with)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgments)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 To-do List Project <a name="about-project"></a>
+
+**This is the To-do List Project in order to show what we have learned so far at HTML, CSS and JS.**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **made with use of github workflow**
+- **made with use of linter checkers**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming Soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Portfolio presentation will be coming soon.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder: https://github.com/mertkantaroglu/To-Do-List-with-Webpack.git
+
+### Install <a name="install"></a>
+
+Install this project with Bash command
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command: Click on Live server option in your IDE
+
+## 👥 Authors <a name="authors"></a>
+
+### 👤 **Mert Kantaroglu**
+
+- GitHub: [@mertkantaroglu](https://github.com/mertkantaroglu/)
+- Twitter: [@aranelluinwe](https://twitter.com/aranelluinwe)
+- LinkedIn: [mertkantaroglu](https://www.linkedin.com/in/mert-kantaroglu/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+For the future features I will add custom styling and also favorite button to the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank for Microverse to provide me learning materials, my peers for helping me out to move forward.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] You must delete or fill the FAQ section and every header from the [README template](https://github.com/microverseinc/readme-template) should be present. `Show your support` is missing please update it.